pallets/providers/Cargo.toml

```diff
@@ -16,14 +16,16 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.0.0", features = [
-	"derive",
+    "derive",
 ], default-features = false }
 scale-info = { version = "2.10.0", default-features = false, features = [
-	"derive",
+    "derive",
 ] }
 
 # Local
-storage-hub-traits = { path = "../../support/traits", version = "0.1.0", default-features = false}
+storage-hub-traits = { workspace = true, default-features = false }
+
+pallet-proofs-dealer = { workspace = true }
```

Comment:
This is the dependency we shouldn't have considering we already depend on `storage-hub-traits`

Reply:
Yes this was put here because of the previous comment about the miss placed `impl TrieVerifier`.

Will remove `pallet-proofs-dealer`.